src/Console/Command/TwigLintCommand.php

```diff
@@ -1,28 +0,0 @@
-<?php
-
-namespace Acquia\Blt\Console\Command;
-
-use Symfony\Bridge\Twig\Command\LintCommand;
-use Symfony\Component\Console\Command\Command;
-use Symfony\Component\Console\Input\InputArgument;
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Input\InputOption;
-use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Yaml\Yaml;
-use Symfony\Component\Yaml\Exception\ParseException;
-
-class TwigLintCommand extends LintCommand
-{
-
-  protected function configure() {
-    $this->addArgument('environment-dir', InputOption::VALUE_REQUIRED);
-    parent::configure();
-  }
-
-  protected function execute(InputInterface $input, OutputInterface $output)
-  {
-    $loader = new \Twig_Loader_Filesystem($input->getArgument('environment-dir'));
-    $this->setTwigEnvironment(new \Twig_Environment($loader));
```

Comment:
If you wanted to go all the way here, you'd actually add Drupal core's Twig extension:
```
    $twig = new \Twig_Environment(NULL, array(
      'debug' => TRUE,
      'cache' => FALSE,
      'autoescape' => 'html',
      'optimizations' => 0
    ));
$twig->addExtension(new TwigExtension($renderer);
```